app/models/foreman_docker/docker.rb

```diff
@@ -69,13 +69,18 @@ def provider_friendly_name
     end
 
     def create_container(args = {})
-      options = vm_instance_defaults.merge(args)
+      options = args.merge(vm_instance_defaults)
```

Comment:
Don't change this order, it's overriding the name so that new containers will be named something like ```foreman_238912931```.